libs/indexer/feature_utils.cpp

```diff
@@ -517,6 +517,13 @@ string FormatCapacity(std::string_view capacity, TypesHolder const & types)
   return {};
 }
 
+string FormatLevel(std::string_view level)
```

Comment:
We try to avoid using `using namespace std` because it may break unity builds.
```suggestion
std::string FormatLevel(std::string_view level)
```